### PR TITLE
Apply last example_ynh  + Fix scripts

### PR DESCRIPTION
**Pull request by yalh76**
capture

### CODE REVIEW

**Comment by yalh76** [comment]
And fix release is working
capture

File: check_process [hunk 1]
```diff
@@ -6,31 +6,28 @@
...
 	; Checks
 		pkg_linter=1
-		setup_sub_dir=0
+		setup_sub_dir=1
 		setup_root=1
 		setup_nourl=0
-		setup_private=0
+		setup_private=1
 		setup_public=1
 		upgrade=1
```
Comment (kay0u) [inline check_process]: Could you add a new line with: `upgrade=1          from_commit==c7d09d7f15a01e85e0354561d737523e1cb18749`.
(and keep the line `upgrade=1` as well).

With this line, the package_check will test the upgrade from the old version to this new one.

Comment (yalh76) [reply check_process]: It's done :)

Comment (yalh76) [reply check_process]: but that was: `upgrade=1 from_commit=c7d09d7f15a01e85e0354561d737523e1cb18749`

Comment (kay0u) [reply check_process]: Oups indeed

**Pull request merged**